src/pages/auth/index.tsx

```diff
@@ -0,0 +1,113 @@
+import "css/auth/auth.css";
+
+export default function Login() {
+  return (
+    <section className="auth-section">
+      <div className="auth">
+        <ul className="tabs">
+          <li
+            id="login"
+            className="tabLinks active"
+            // onClick={() => console.log("clicked")}
+          >
+            Login
+          </li>
+          <li
+            id="signup"
+            className="tabLinks"
+            // onClick="openTab(event, 'signup')"
```

Comment:
This one also

Reply:
This was done in js so I had to convert it to rect, I'm going to work on auth in the upcoming tickets, thanks for the review